src/main/java/uk/nhs/digital/nhsconnect/nhais/service/RegistrationConsumerService.java

```diff
@@ -131,8 +133,7 @@ private boolean hasAlreadyBeenProcessed(Transaction transaction) {
 
     private MeshMessage buildRecepMeshMessage(String edifactRecep) {
         return new MeshMessage()
-            // TODO: determine ODS code: probably via ENV? or should it be taken from incoming mesh message?
-            .setHaTradingPartnerCode("ods123")
+            .setHaTradingPartnerCode(meshConfig.getMailboxId())
```

Comment:
what about this HatradingPartnerCode?

Reply:
Pass `Interchange` to this method and take sender from it